src/main/java/org/fundacionjala/coding/kevincristhian/Movie.java

```diff
@@ -0,0 +1,76 @@
+package org.fundacionjala.coding.kevincristhian;
+
+/**
+ * father class for movies.
+ */
+public abstract class Movie {
+    private String title;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "title" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)